### ISSUE TITLE
Confusing wording in Encryption page

### ISSUE THREAD

**Issue by phoebegoh** [NetAppDocs/occm#152]
Page: [Encrypting volumes with NetApp Volume Encryption](https://docs.netapp.com/us-en/occm/task_encrypting_volumes.html)

A couple of requests to clarify the wording around these sections:

> You need to set up NetApp Volume Encryption from the CLI. You can then use either the CLI or System Manager to enable encryption on specific volumes. Cloud Manager does not support NetApp Volume Encryption.

Could you reword this to indicate that Cloud Manager does not support NVE via the OCCM GUI. A lot of people associate OCCM and CVO and it is confusing as written.

> Cloud Volumes ONTAP encryption is no longer supported in AWS
> Data-at-rest encryption of aggregates using external key managers is no longer supported.

Can you clarify that data-at-rest-encryption of the system is still supported via AWS KMS. The first line is very confusing (since encryption IS supported, just not via the previous method)

**Comment by netapp-bcammett** [comment]
Hi Phoebe,

Thank you for your feedback. Here are proposed changes. Let me know what you think.

**Comment 1**

You need to set up NetApp Volume Encryption from the ONTAP CLI. You can then use the CLI or System Manager to enable encryption on specific volumes. Cloud Manager does not support NetApp Volume Encryption from its user interface and from its APIs.

**Comment 2**

Data-at-rest encryption of aggregates using external key managers is no longer supported. If you are currently using this feature and you want to upgrade, you must launch a new 9.5 system and then [replicate data](https://docs.netapp.com/us-en/occm/task_replicating_data.html) to that system.

Data-at-rest encryption is still supported using other methods. You can encrypt data by using NetApp Volume Encryption or by using the AWS Key Management Service (KMS). [Learn more about encryption of data at rest](https://docs.netapp.com/us-en/occm/concept_security.html).

Thanks again,
Ben

**Comment by phoebegoh** [comment]
Much clearer, thankyou! I like the links to other sections too. Thank you Ben.
